<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="7" total="15" passed="8">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="99971" started-at="2018-11-20T13:29:34Z" finished-at="2018-11-20T13:31:14Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="99971" started-at="2018-11-20T13:29:34Z" finished-at="2018-11-20T13:31:14Z">
      <class name="com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher">
        <test-method status="PASS" signature="setup1()[pri:0, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="setup1" is-config="true" duration-ms="9717" started-at="2018-11-20T18:59:34Z" finished-at="2018-11-20T18:59:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup1 -->
        <test-method status="PASS" signature="TCED14151()[pri:1, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14151" duration-ms="13472" started-at="2018-11-20T18:59:44Z" finished-at="2018-11-20T18:59:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14151 -->
        <test-method status="PASS" signature="TCED14152()[pri:2, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14152" duration-ms="164" started-at="2018-11-20T18:59:57Z" finished-at="2018-11-20T18:59:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14152 -->
        <test-method status="PASS" signature="TCED14153()[pri:3, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14153" duration-ms="1721" started-at="2018-11-20T18:59:57Z" finished-at="2018-11-20T18:59:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14153 -->
        <test-method status="PASS" signature="TCED14154()[pri:4, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14154" duration-ms="16513" started-at="2018-11-20T18:59:59Z" finished-at="2018-11-20T19:00:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14154 -->
        <test-method status="PASS" signature="TCED14155()[pri:5, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14155" duration-ms="20244" started-at="2018-11-20T19:00:16Z" finished-at="2018-11-20T19:00:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14155 -->
        <test-method status="PASS" signature="TCED14156()[pri:6, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14156" duration-ms="12272" started-at="2018-11-20T19:00:36Z" finished-at="2018-11-20T19:00:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14156 -->
        <test-method status="PASS" signature="TCED14157()[pri:7, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14157" duration-ms="2309" started-at="2018-11-20T19:00:48Z" finished-at="2018-11-20T19:00:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14157 -->
        <test-method status="PASS" signature="TCED14158()[pri:8, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14158" duration-ms="10368" started-at="2018-11-20T19:00:50Z" finished-at="2018-11-20T19:01:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14158 -->
        <test-method status="FAIL" signature="TCED14159()[pri:9, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14159" duration-ms="5002" started-at="2018-11-20T19:01:01Z" finished-at="2018-11-20T19:01:06Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failed to download document which has extension .CSV expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failed to download document which has extension .CSV expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher.TCED14159(StudentAssessmentAdministrationResultsbyTeacher.java:387)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14159 -->
        <test-method status="FAIL" signature="TCED14160()[pri:10, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14160" duration-ms="25" started-at="2018-11-20T19:01:06Z" finished-at="2018-11-20T19:01:06Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Tried to run command without establishing a connection
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'ANS-A', ip: '192.168.10.124', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 57.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 5716, moz:profile: C:\Users\Ans-A\AppData\Loca..., moz:webdriverClick: false, pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 10.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 1e884650-d9cb-48d0-9f7e-6e028d4c71a5
*** Element info: {Using=xpath, value=//*[@id='ctl00_tdContentCell']/table/tbody/tr[3]/td/table[4]/tbody/tr[2]/td/table[1]/tbody/tr[2]/td/table/tbody/tr[1]/td[1]}]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Tried to run command without establishing a connection
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'ANS-A', ip: '192.168.10.124', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 57.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 5716, moz:profile: C:\Users\Ans-A\AppData\Loca..., moz:webdriverClick: false, pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 10.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 1e884650-d9cb-48d0-9f7e-6e028d4c71a5
*** Element info: {Using=xpath, value=//*[@id='ctl00_tdContentCell']/table/tbody/tr[3]/td/table[4]/tbody/tr[2]/td/table[1]/tbody/tr[2]/td/table/tbody/tr[1]/td[1]}
	at org.testng.Assert.fail(Assert.java:94)
	at com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher.TCED14160(StudentAssessmentAdministrationResultsbyTeacher.java:451)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14160 -->
        <test-method status="FAIL" signature="TCED14161()[pri:11, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14161" duration-ms="4019" started-at="2018-11-20T19:01:06Z" finished-at="2018-11-20T19:01:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failed to download document which has extension .PDF expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failed to download document which has extension .PDF expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher.TCED14161(StudentAssessmentAdministrationResultsbyTeacher.java:479)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14161 -->
        <test-method status="FAIL" signature="TCED14162()[pri:12, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14162" duration-ms="21" started-at="2018-11-20T19:01:10Z" finished-at="2018-11-20T19:01:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Tried to run command without establishing a connection
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'ANS-A', ip: '192.168.10.124', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 57.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 5716, moz:profile: C:\Users\Ans-A\AppData\Loca..., moz:webdriverClick: false, pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 10.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 1e884650-d9cb-48d0-9f7e-6e028d4c71a5
*** Element info: {Using=xpath, value=//*[@id='ctl00_MainContent_ddlStudentGroups']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Tried to run command without establishing a connection
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'ANS-A', ip: '192.168.10.124', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 57.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 5716, moz:profile: C:\Users\Ans-A\AppData\Loca..., moz:webdriverClick: false, pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 10.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 1e884650-d9cb-48d0-9f7e-6e028d4c71a5
*** Element info: {Using=xpath, value=//*[@id='ctl00_MainContent_ddlStudentGroups']}
	at org.testng.Assert.fail(Assert.java:94)
	at com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher.TCED14162(StudentAssessmentAdministrationResultsbyTeacher.java:550)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14162 -->
        <test-method status="FAIL" signature="TCED14163()[pri:13, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14163" duration-ms="2022" started-at="2018-11-20T19:01:10Z" finished-at="2018-11-20T19:01:12Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Tried to run command without establishing a connection
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'ANS-A', ip: '192.168.10.124', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 57.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 5716, moz:profile: C:\Users\Ans-A\AppData\Loca..., moz:webdriverClick: false, pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 10.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 1e884650-d9cb-48d0-9f7e-6e028d4c71a5
*** Element info: {Using=xpath, value=//*[@id='ctl00_MainContent_ddlStudentGroups']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Tried to run command without establishing a connection
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'ANS-A', ip: '192.168.10.124', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 57.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 5716, moz:profile: C:\Users\Ans-A\AppData\Loca..., moz:webdriverClick: false, pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 10.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 1e884650-d9cb-48d0-9f7e-6e028d4c71a5
*** Element info: {Using=xpath, value=//*[@id='ctl00_MainContent_ddlStudentGroups']}
	at org.testng.Assert.fail(Assert.java:94)
	at com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher.TCED14163(StudentAssessmentAdministrationResultsbyTeacher.java:587)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14163 -->
        <test-method status="FAIL" signature="TCED14164()[pri:14, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14164" duration-ms="2037" started-at="2018-11-20T19:01:12Z" finished-at="2018-11-20T19:01:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Tried to run command without establishing a connection
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'ANS-A', ip: '192.168.10.124', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 57.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 5716, moz:profile: C:\Users\Ans-A\AppData\Loca..., moz:webdriverClick: false, pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 10.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 1e884650-d9cb-48d0-9f7e-6e028d4c71a5
*** Element info: {Using=css selector, value=frame[name='rwTestSummery'],iframe[name='rwTestSummery']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Tried to run command without establishing a connection
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'ANS-A', ip: '192.168.10.124', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 57.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 5716, moz:profile: C:\Users\Ans-A\AppData\Loca..., moz:webdriverClick: false, pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 10.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 1e884650-d9cb-48d0-9f7e-6e028d4c71a5
*** Element info: {Using=css selector, value=frame[name='rwTestSummery'],iframe[name='rwTestSummery']}
	at org.testng.Assert.fail(Assert.java:94)
	at com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher.TCED14164(StudentAssessmentAdministrationResultsbyTeacher.java:610)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14164 -->
        <test-method status="FAIL" signature="TCED14165()[pri:15, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="TCED14165" duration-ms="6" started-at="2018-11-20T19:01:14Z" finished-at="2018-11-20T19:01:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Tried to run command without establishing a connection
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'ANS-A', ip: '192.168.10.124', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 57.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 5716, moz:profile: C:\Users\Ans-A\AppData\Loca..., moz:webdriverClick: false, pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 10.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 1e884650-d9cb-48d0-9f7e-6e028d4c71a5]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Tried to run command without establishing a connection
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'ANS-A', ip: '192.168.10.124', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 57.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 5716, moz:profile: C:\Users\Ans-A\AppData\Loca..., moz:webdriverClick: false, pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 10.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 1e884650-d9cb-48d0-9f7e-6e028d4c71a5
	at org.testng.Assert.fail(Assert.java:94)
	at com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher.TCED14165(StudentAssessmentAdministrationResultsbyTeacher.java:695)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TCED14165 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher@271053e1]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-11-20T19:01:14Z" finished-at="2018-11-20T19:01:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.analyzeByScore.StudentAssessmentAdministrationResultsbyTeacher -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
